charts/bootstrap/templates/redhat-portal-credentials-secret.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.rhsm.portal.secretName }}
+  namespace: {{ .Values.rhsm.portal.namespace }}
```

```
```suggestion
  namespace: {{ .Values.rhsm.portal.namespace }}
```
```suggestion
  namespace: {{ .Values.rhsm.namespace }}
```
Take a look at:
https://github.com/redhat-cop/rhel-edge-automation-arch/blob/0c9e4dfe95547c230807546ed5a30a6ed42cd451/charts/bootstrap/values.yaml#L17-L18